JSON_Javascript/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Exercise JS</title>
+  <link rel="stylesheet" href="bootstrap/bootstrap.min.css">
+  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+    <nav class="navigation">
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Shop</a></li>
+        <li><a href="#">Product</a></li>
+        <li><a href="#">About us</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <button type="button" class="btn-cart" id="button">
+      <img src="images/empty-cart-icon.png">
+      <span id="display">0</span>
+   </button>
+  </header>
+  <main>
+    <div class="container">
+      <div class="row" id="list-Product">
+
```

Comment:
- Remove new line and for it on 1 line.
- Name id `product-list` 
- Never use the name like this: `-Product` don't uppercase first letter after the hyphen.